### PR TITLE
Fix Omnisearch to generate paths

### PR DESCRIPTION
**Pull request by CarsonF**
Also a couple other things.

Is everyone ok with `app->path()`? (3658157)

### CODE REVIEW

File: src/Application.php [hunk 1]
```diff
@@ -569,4 +566,20 @@ public function getVersion($long = true)
...
+     *
+     * @param string $route      The name of the route
+     * @param array  $parameters An array of parameters
+     *
+     * @return string The generated path
+     */
+    public function path($route, $parameters = array())
+    {
+        return $this['url_generator']->generate($route, $parameters);
+    }
```
Comment (GawainLynch) [inline src/Application.php]: Like the idea, not sure if the name isn't too ambiguous though (per IRC discussion)

Comment (CarsonF) [inline src/Application.php]: This is the same as what `UrlGeneratorTrait` and `Library` call. But yes I agree. 
I thought about changing it to `generatePath`

Comment (GawainLynch) [inline src/Application.php]: That is actually pretty valid reasoning.

Comment (bobdenotter) [inline src/Application.php]: Yes, I like `generatePath` too. :+1:

**Comment by bobdenotter** [comment]
@CarsonF Don't forget to change the name to `generatePath()`;-)

**Comment by GawainLynch** [comment]
Travis failures are due to timeouts on `http://loripsum.net/` again.

**Comment by bobdenotter** [comment]
Choo Choo! :steam_locomotive: :train:  :train:  :train:  :train:

**Pull request merged**